src/arkode/arkode_arkstep.c

```diff
@@ -137,16 +128,14 @@ void* ARKStepCreate(ARKRhsFn fe, ARKRhsFn fi, realtype t0, N_Vector y0,
   /* If an implicit component is to be solved, create default Newton NLS object */
   step_mem->ownNLS = SUNFALSE;
   if (step_mem->implicit)  {
-    NLS = SUNNonlinSol_Newton(y0, ark_mem->sunctx);
+    NLS = SUNCheckCallLastErrNoRet(SUNNonlinSol_Newton(y0, ARK_SUNCTX));
```

Comment:
I must have missed where `ARK_SUNCTX` is defined -- that doesn't seem to be anywhere in this file, nor did I see it in the header files.

As I mentioned earlier, I am against readability constants/macros, and this looks like one.  Was there something improper/unusable about directly referencing the SUNContext object, `ark_mem->sunctx`?

Reply:
Indeed this is an instance of a shortcut macro. We can change it to `ark_mem->sunctx`.